nss-day-cohort-48/the-worst-site-ever-eliminatorz
# Finished madeline.html and madeline.css

I finished adding all of the content and styling to madeline.html and madeline.css.  I also added a mecPageImages directory.

##steps to test

1. pull this branch
2. open madeline.html in chrome
3. page should now display ads and a portfolio section
4. portfolio links should link to websites listed
5. font should display kashuan script
